tests/ast/data/DataFrame.agg.test

```diff
@@ -239,49 +206,39 @@ body {
                   fn {
                     builtin_fn {
                       name {
-                        fn_name_flat {
-                          name: "col"
+                        name {
```

Comment:
This looks like it's nested twice -- is this intentional?

Reply:
Yes, the outer `name` is of type `SpNameRef` and comes from the trait `FnNameRefExpr`, and the inner `name` is the field of the `SpNameRef`, of type `SpName`.